build/helm/cloudcore/crds/cluster_objectsync_v1alpha1.yaml

```diff
@@ -0,0 +1,75 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: clusterobjectsyncs.reliablesyncs.kubeedge.io
```

Comment:
what's difference of these defined files and files in https://github.com/kubeedge/kubeedge/tree/master/build/crds/devices

Reply:
No different, just copy it here to install them with helm.